src/fugw/solvers/dense.py

```diff
@@ -20,7 +21,7 @@ class FUGWSolver(BaseSolver):
     """Solver computing dense solutions"""
 
     def local_biconvex_cost(
-        self, pi, transpose, data_const, tuple_weights, hyperparams
+        self, pi, transpose, data_const, tuple_weights, hyperparams, divergence
```

Comment:
We should probably try to fit `divergence` in `hyperparams`, because all current variables which define the shape of loss function are in `hyperparams` :slightly_smiling_face:

Reply:
I agree, I will put all in `hyperparams`